### PR TITLE
feat: #786 move tags under description

### PR DESCRIPTION
**Pull request by Anok666**

### CODE REVIEW

**Comment by Marcelinna** [comment]
Please change the commit description - this PR is about show button for keywords

**Comment by Marcelinna** [comment]
Which rules do you use for the "show more" button for keywords?
image

**Comment by Marcelinna** [comment]
I see the problem with pagination
I don't see the problem on the beta instance
image
image

File: ui/apps/ui/src/app/layouts/secondary-tags/secondary-tags.component.ts [hunk 1]
```diff
@@ -78,6 +84,48 @@ export class SecondaryTagsComponent implements OnChanges {
...
+  temp: IValueWithLabel[] = [];
+
+  constructor(private cdr: ChangeDetectorRef) {}
+
+  ngAfterViewInit() {
+    this.cdr.detectChanges();
+  }
+
+  ngAfterContentChecked(): void {
+    this.cdr.detectChanges();
```
Comment (tomaszandel) [inline ui/apps/ui/src/app/layouts/secondary-tags/secondary-tags.component.ts:98]: smart but.. it wont call onChanges, onChanges called only when inputparams changed, even if you call detectChanges still onChanges not called, i would still implement onChanges interface.. Please double check if what you did here is intentional or a mistake ;)

Comment (Anok666) [reply ui/apps/ui/src/app/layouts/secondary-tags/secondary-tags.component.ts:98]: I made a mistake. Old unused code

**Comment by Anok666** [comment]
> URL

I cant repeat this behavior / bug.

**Pull request closed without merging**